src/libtriton/includes/triton/instruction.hpp

```diff
@@ -107,6 +107,9 @@ namespace triton {
         //! True if this instruction updartes flags. Mainly used for AArch64 instruction like ADDS.
         bool updateFlag;
 
+        //! True if this is a Thumb instruction.
+        bool thumb;
```

Comment:
`isThumb()` are implemented in `Instruction` and `Arm32Cpu`. Do we really have to use these two methods? Can we not just use one of them?

Reply:
A quick note about both functions. `isThumb()` in `Arm32Cpu` is used to determine the current execution mode, while in `Instruction` it indicates whether the instruction itself is a Thumb instruction. In the first case, it is used in `Arm32Semantics` when switching modes (check, for instance, `exchangeInstructionSet`). On the other hand, `isThumb()` in `Instruction` is not currently used, however, it could be useful when doing an analysis that depends on the type of the instruction.

Reply:
Commit 6b44a7817cbde713c2a89a4293375b902b8e9360 address this.

Reply:
Commit 2fc6e342b3d34e6a79c454face6b0153d2839628 add a note to clarify this.